src/index.js

```diff
@@ -6,6 +6,7 @@ const Xapian = imports.gi.Xapian;
 
 const SEQUENCE_NUMBER_VALUE_NO = 0;
 const PUBLISHED_DATE_VALUE_NO = 1;
+const ALPHABETICAL_VALUE_NO = 2;
```

Comment:
It occurred to me that it would be good to import EknContent and share these constants. Definitely out of scope for this pull request, though.